frontend/sentry.edge.config.ts

```diff
@@ -0,0 +1,7 @@
+import * as Sentry from '@sentry/nextjs';
+
+const SENTRY_DSN = process.env.NEXT_PUBLIC_SENTRY_DSN;
+
+Sentry.init({
+   dsn: SENTRY_DSN,
```

Comment:
I don't think we're using vercel edge, so this config won't be used. However, I understand we might have added this to silence a sentry warning during the build process.

If we're going to add this file, I'd like us to add this tag. If we ever start using vercel edge, the config will be ready to go.

```suggestion
   dsn: SENTRY_DSN,
      initialScope: {
      tags: {
         'next.runtime': 'edge',
      },
   },
```